modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1,819 +0,0 @@
-import { describe, it } from '@ephox/bedrock-client';
-import { Arr } from '@ephox/katamari';
-import { assert } from 'chai';
-
-import Env from 'tinymce/core/api/Env';
-import { BlobCache } from 'tinymce/core/api/file/BlobCache';
-import DomParser from 'tinymce/core/api/html/DomParser';
-import AstNode from 'tinymce/core/api/html/Node';
-import Schema from 'tinymce/core/api/html/Schema';
-import HtmlSerializer from 'tinymce/core/api/html/Serializer';
-
-describe('browser.tinymce.core.html.DomParserTest', () => {
```

Comment:
This test shouldn't have been deleted, or at least not entirely as some of it applies to validation and "fixing" nodes. Please revert this and either skip it or comment it out instead so we can determine what should be kept.

- [ ] Revert deleting this test and skip/comment instead

Reply:
Back, with Jiras to fix it up